cookbooks/heat_flow/heat-flow.prm

```diff
@@ -0,0 +1,122 @@
+# A model of spreading at a mid-ocean ridge.
```

Comment:
I copied the composition-reaction cookbook to use as a starting point, and that is also why it still has some of those old comments. But I think it does not actually make sense to read that first, since (1) it's less earth-like (nondimensional, in terms of boundary conditions, etc.) and it is more complicated in that it has the reaction part that is irrelevant here. So I am just going to try and remove all those references. I think since these are teaching cookbooks, they should be able to stand on their own anyway, and my students never looked at any of the other cookbooks before this one.

Reply:
Agree that it is best to have the teaching cookbooks stand on their own (as much as possible). Thanks for going through and removing the old references/comments.